internal/hscan/structmap.go

```diff
@@ -78,10 +79,14 @@ type StructValue struct {
 	value reflect.Value
 }
 
-func (s StructValue) Scan(key string, value string) error {
+func (s StructValue) Scan(key string, value string) (err error) {
 	field, ok := s.spec.m[key]
-	if !ok {
-		return nil
```

Comment:
This is minor but using early returns significantly improves readability - https://danp.net/2015/11/02/reducing-go-nesting.html